components/director/internal/domain/runtime/repository.go

```diff
@@ -267,11 +259,21 @@ func (r *pgRepository) GetOldestForFilters(ctx context.Context, tenant string, f
 	orderByParams := repo.OrderByParams{repo.NewAscOrderBy("creation_timestamp")}
 
 	var runtimeEnt Runtime
-	if err := r.singleGetter.Get(ctx, tenant, additionalConditions, orderByParams, &runtimeEnt); err != nil {
+	if err := r.singleGetter.Get(ctx, resource.Runtime, tenant, additionalConditions, orderByParams, &runtimeEnt); err != nil {
 		return nil, err
 	}
 
-	runtimeModel := runtimeEnt.ToModel()
+	runtimeModel := r.conv.FromEntity(&runtimeEnt)
 
 	return runtimeModel, nil
 }
+
+func (r *pgRepository) multipleFromEntities(entities RuntimeCollection) []*model.Runtime {
```

Comment:
I know that most of the times such methods are added to the converter, however in my opinion the converter should be aware of how to convert an entity to model and vice versa. 

The main difference is that if you add it to the converter, then this method becomes part of the interface of the converter -> should be mocked and therefore used in all tests as black box. I don't think that this is conceptually what a converter should do and I would like to be able to configure each of the FromEntity invocations with different mock setup in order to test different scenarios (this is not critical in this example but in others it is).

That's personal preference, if you think that it is better (it will be more consistent with the other entities) we can move it to the converter I am also fine with it.